### DISCUSSION TITLE
Reboot after RFSend (Restart Reason: Software Watchdog)

### DISCUSSION THREAD

**Discussion by ghost** [arendst/Tasmota#13290] [Q&A] [answered]
I connected an RF transmitter to GPIO14 on NodeMCU LoLin v3. Whenever I send command RfSend (e.g. RfSend 0x24B7C4, 24, 1, 399), the device will reboot, with restart reason Software Watchdog. The signal was sent successfully, though, because another device was able to receive the signal.

It's on 9.5.0.1(sensors). What could be wrong?

**Comment by barbudor** [comment, marked as answer]
Does the reboot occurs also when the RF transmitter is not connected?

**Reply by ghost** [reply]
It reboots even without RF transmitter connected. Same behavior when sending `rfsend `from console and from MQTT publish.

**Reply by barbudor** [reply]
You command is wrong and missing 1 number: https://tasmota.github.io/docs/Commands/#rf-transciever
You requested 399 repeats which are taking too long. This is why the watchdog triggers.
So indeed RfSend should manage the watchdog between repeats but however when entering `RfSend 0x24B7C4, 24, 1, 10, 399` everything is fine

**Reply by ghost** [reply]
Oh wow. You were right. Missing a number. It no longer reboots now, and returns result `{"RFSend":"Done"}`. Thanks.
I got it from a sniffed code using RF receiver. ~~Obviously, it was the wrong code sniffed (cheap receiver, bad reviews online).~~ Sniffed code does not have repeat, and now I know it needs to be added to send.
